### ISSUE TITLE
Panic while getting unsafe counters for strum crate

### ISSUE THREAD

**Issue by bertptrs** [rust-secure-code/cargo-geiger#105]
Trying to analyse anything that depends on the `strum` crate results in a panic while analysing it. Below is a minimal Cargo.toml file that shows the issue.

```toml
[package]
name = "geiger-crash"
version = "0.1.0"
edition = "2018"

[dependencies]
strum = "0.18.0"
```

This results in the following backtrace:

```
    Checking strum v0.18.0
    Checking geiger-crash v0.1.0 (/home/bert/Downloads/geiger-crash)
    Finished dev [unoptimized + debuginfo] target(s) in 0.71s
Failed to parse file: /home/bert/.cargo/registry/src/github.com-1ecc6299db9ec823/strum-0.18.0/src/lib.rs, Syn(Error("expected `:`"), "/home/bert/.cargo/registry/src/github.com-1ecc6299db9ec823/strum-0.18.0/src/lib.rs") 
    Scanning done

Metric output format: x/y
    x = unsafe code used by the build
    y = total unsafe code found in the crate

Symbols: 
    :) = No `unsafe` usage found, declares #![forbid(unsafe_code)]
    ?  = No `unsafe` usage found, missing #![forbid(unsafe_code)]
    !  = `unsafe` usage found

Functions  Expressions  Impls  Traits  Methods  Dependency

0/0        0/0          0/0    0/0     0/0      ?  geiger-crash 0.1.0
thread 'main' panicked at 'Failed to get unsafe counters for package: strum v0.18.0', cargo-geiger/src/cli.rs:366:25
stack backtrace:
   0:     0x556661a42e75 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hfbfe9ad65a0640ca
   1:     0x556661a6b5ec - core::fmt::write::hb409a30975c0f9f5
   2:     0x556661a3d383 - std::io::Write::write_fmt::h7ee7c68a0296cf9a
   3:     0x556661a46c20 - std::panicking::default_hook::{{closure}}::h5e5a680f03f4adf9
   4:     0x556661a4697c - std::panicking::default_hook::hb8f06365ed415b3d
   5:     0x556661a47344 - std::panicking::rust_panic_with_hook::h1e03621439587007
   6:     0x556661a46e5b - rust_begin_unwind
   7:     0x556661a46dcb - std::panicking::begin_panic_fmt::hc43e1369214a6caa
   8:     0x5566607cc15e - cargo_geiger::cli::run_scan_mode_default::{{closure}}::hafc944e15c547772
                               at cargo-geiger/src/cli.rs:366
   9:     0x5566607d8cb2 - core::option::Option<T>::unwrap_or_else::ha81cfaf0ebe95922
                               at /build/rust/src/rustc-1.43.1-src/src/libcore/option.rs:428
  10:     0x5566607ab41a - cargo_geiger::cli::run_scan_mode_default::h4fb274a796135f8d
                               at cargo-geiger/src/cli.rs:361
  11:     0x5566607c9e53 - cargo_geiger::real_main::h3e1c79b2faa1ad85
                               at cargo-geiger/src/main.rs:320
  12:     0x5566607ca333 - cargo_geiger::main::hc6c05f797b1d91c3
                               at cargo-geiger/src/main.rs:342
  13:     0x5566607b810b - std::rt::lang_start::{{closure}}::h2d240e20af9249f8
                               at /build/rust/src/rustc-1.43.1-src/src/libstd/rt.rs:67
  14:     0x556661a46d33 - std::panicking::try::do_call::hdcef0c1305ca8214
  15:     0x556661a51397 - __rust_maybe_catch_panic
  16:     0x556661a4784c - std::rt::lang_start_internal::h0354be261626f155
  17:     0x5566607b80e7 - std::rt::lang_start::h230dbfb9ebb8f1f6
                               at /build/rust/src/rustc-1.43.1-src/src/libstd/rt.rs:67
  18:     0x5566607ca66a - main
  19:     0x7f214605d002 - __libc_start_main
  20:     0x5566607800fe - _start
  21:                0x0 - <unknown>
```

**Comment by repi** [comment]
Ran into this panic with `strum` in our codebase as well, making cargo-geiger not really work as it well aborts out. Would be great to fix this or at least not panic on it (warning is fine)

**Comment by anderejd** [comment]
I'll try to find some time later today :)

**Comment by anderejd** [comment]
Released as part of 0.10.2.

**Comment by bertptrs** [comment]
Thank you! Time to update my package then.
